api/api.go

```diff
@@ -106,7 +108,10 @@ func (h *httpHandler) initRouter() {
 			{"/services", []string{"partial-id", epPIDQr}, procGetServices},
 			{"/services", nil, procGetServices},
 			{"/services/" + epID, nil, procGetService},
-			{"/services/" + epID + "/backend", nil, procGetContainers},
+			{"/services/" + epID + "/backend", nil, procGetSandbox},
```

Comment:
We still need a URL for the Join/Leave in the POST/DELETE list.
If we do so, then I feel we should provide the GET for the same URL.